test/utils/allInvalid.js

```diff
@@ -1,7 +1,8 @@
-import validate from '../../src/index'
 import chalk from 'chalk'
 import util from 'util'
 
+const validate = require('../../src/index').validate
```

Comment:
The way I've structured this is that `validateRoot` supports the array of configurations and is the single entry point for `validate-config.js`, as seen [here](https://github.com/js-dxtools/webpack-validator/pull/119/files#diff-ee0e20c822ec0f4af6bdb5b889841c0fR2).

`validate` is simply used for validating the individual properties of the configs, which is done through this utility methods in `allInvalid.js` and `allValid.js`.